### ISSUE TITLE
binary-expression-operand-order wrong for all-numeric expressions

### ISSUE THREAD

**Issue by sebek64** [palantir/tslint#2866]
### Bug Report

- __TSLint version__: 5.4.2
- __TypeScript version__: 5.3.3
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
private static readonly x = 1000 * 3600 * 24 * 7;
```

with `tslint.json` configuration:

```json
"binary-expression-operand-order": true
```

#### Actual behavior

Produces an error:

Literal expression should be on the right-hand side of a binary expression.

#### Expected behavior

Should not produce an error.
